content/set-up/clients/client-credentials-client-procedure.md

```diff
@@ -20,37 +20,37 @@ To add a client-credentials client:
 
```

Comment:
Same feedback about the dropdown menu step in the previous topic--we need a locater for the dropdown menu, and the options label needs updating to "Client-Credentials Clients".

Reply:
Feel free to ignore this one based on what I said earlier.